### ISSUE TITLE
Current temperature for v2 API is wrong

### ISSUE THREAD

**Issue by fury182** [chubin/wttr.in#683]
The temperature for the v1 API is correct and matches that from the official sources but the current temperature from v2 is very different and often seems "locked" all the time to the same value. Is this temperature retrieved from other sources?

Thanks

**Comment by chubin** [comment]
@fury182 No, actually it should not. They use the same data source. Probably it is a caching issue.
Could you please verify, if you are still experiencing the same problem?
